docs/spark/how-to-guides/windows-instructions.md

```diff
@@ -0,0 +1,253 @@
+        -a----        3/16/2019  12:00 AM          23552 Microsoft.Spark.Worker.xml
+        -a----        3/16/2019  12:00 AM         332363 Microsoft.Spark.xml
+        ------------------------------------------- More framework files -------------------------------------
+      ```     
+
+#### Using .NET Core CLI for .NET Core
+
+> [!NOTE]
+> We are currently working on automating .NET Core builds for Spark .NET. Until then, we appreciate your patience in performing some of the steps manually.
+
+  1. Build the worker.
+
+      ```powershell
+      cd C:\github\dotnet-spark\src\csharp\Microsoft.Spark.Worker\
+      dotnet publish -f netcoreapp2.1 -r win10-x64
+      ```
+      
+      Sample console output:
+
+      ```powershell
+      PS C:\github\dotnet-spark\src\csharp\Microsoft.Spark.Worker> dotnet publish -f netcoreapp2.1 -r win10-x64
+      Microsoft (R) Build Engine version 16.0.462+g62fb89029d for .NET Core
+      Copyright (C) Microsoft Corporation. All rights reserved.
+
+        Restore completed in 299.95 ms for C:\github\dotnet-spark\src\csharp\Microsoft.Spark\Microsoft.Spark.csproj.
+        Restore completed in 306.62 ms for C:\github\dotnet-spark\src\csharp\Microsoft.Spark.Worker\Microsoft.Spark.Worker.csproj.
+        Microsoft.Spark -> C:\github\dotnet-spark\artifacts\bin\Microsoft.Spark\Debug\netstandard2.0\Microsoft.Spark.dll
+        Microsoft.Spark.Worker -> C:\github\dotnet-spark\artifacts\bin\Microsoft.Spark.Worker\Debug\netcoreapp2.1\win10-x64\Microsoft.Spark.Worker.dll
+        Microsoft.Spark.Worker -> C:\github\dotnet-spark\artifacts\bin\Microsoft.Spark.Worker\Debug\netcoreapp2.1\win10-x64\publish\
+      ```    
+
+  2. Build the samples.
+
+      ```powershell
+      cd C:\github\dotnet-spark\examples\Microsoft.Spark.CSharp.Examples\
+      dotnet publish -f netcoreapp2.1 -r win10-x64
+      ```
+   
+      Sample console output:
+
+      ```powershell
+      PS C:\github\dotnet-spark\examples\Microsoft.Spark.CSharp.Examples> dotnet publish -f netcoreapp2.1 -r win10-x64
+      Microsoft (R) Build Engine version 16.0.462+g62fb89029d for .NET Core
+      Copyright (C) Microsoft Corporation. All rights reserved.
+
+        Restore completed in 44.22 ms for C:\github\dotnet-spark\src\csharp\Microsoft.Spark\Microsoft.Spark.csproj.
+        Restore completed in 336.94 ms for C:\github\dotnet-spark\examples\Microsoft.Spark.CSharp.Examples\Microsoft.Spark.CSharp.Examples.csproj.
+        Microsoft.Spark -> C:\github\dotnet-spark\artifacts\bin\Microsoft.Spark\Debug\netstandard2.0\Microsoft.Spark.dll
+        Microsoft.Spark.CSharp.Examples -> C:\github\dotnet-spark\artifacts\bin\Microsoft.Spark.CSharp.Examples\Debug\netcoreapp2.1\win10-x64\Microsoft.Spark.CSharp.Examples.dll
+        Microsoft.Spark.CSharp.Examples -> C:\github\dotnet-spark\artifacts\bin\Microsoft.Spark.CSharp.Examples\Debug\netcoreapp2.1\win10-x64\publish\
+      ```     
+
+## Run samples
```

Comment:
Whatever the final header ends up being for "Build a .NET samples application," I'd reword this header to be more consistent, i.e. "Run the .NET for Spark samples"